src/main/java/uk/ac/ebi/eva/commons/mongodb/filter/VariantEntityRepositoryConsequenceTypeFilter.java

```diff
@@ -29,9 +27,9 @@
 
     public VariantEntityRepositoryConsequenceTypeFilter(List<String> consequenceType) {
         super(FIELD,
-              consequenceType.stream()
-                             .map(c -> Integer.parseInt(c.replaceAll("[^\\d.]", ""), 10))  // parse integer from string
-                             .collect(Collectors.toList()),
-              VariantEntityRepository.RelationalOperator.IN);
+                consequenceType.stream()
+                        .map(c -> Integer.parseInt(c.replaceAll("[^\\d.]", ""), 10))  // parse integer from string
+                        .collect(Collectors.toList()),
+                RelationalOperator.IN);
```

Comment:
what changed here? only the indentation? the agreed style is alignment, not 2 tabs. if you used 2 tabs in all this pr, I'd suggest reviewing the style guide